app/client/components/NavbarAfter.tsx

```
@@ -121,6 +152,31 @@ export default function NavbarAfter() {
           )}
         </div>
       </div>
+      {searchTerm && (
+        <div className="absolute top-14 right-[210px] z-30 border-2 border-red-400 mt-1 bg-white rounded-xl shadow-lg w-[530px] 2xl:right-[575px]">
+          {filteredCourses.slice(0, 3).length > 0 ? (
+            filteredCourses.slice(0, 3).map((course) => (
+              <Link href={`/client/video/search/${encodeURIComponent(course.title.replace(/ /g, '-'))}`}>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)